.github/workflows/test.yml

```diff
@@ -1,19 +1,34 @@
 name: aiosmtpd CI
 
 on:
+  # This is for direct commit to master
   push:
     branches: [ master ]
+  # This is for PRs
   pull_request:
     branches: [ master ]
+  # Manual/on-demand
   workflow_dispatch:
+  release:
+    types: [ created, edited, published, prereleased, released ]
 
 jobs:
   test:
-    runs-on: ubuntu-latest
-    continue-on-error: true
```

Comment:
A bit confused at `continue-on-error` description in GA's docs ... seems like the whole workflow can still succeed if any matrix fail?

So I used a more explicit "allow one matrix to fail" setting instead: `fail-fast` (see below)